subsystems/intakesubsystem.py

```diff
@@ -161,13 +160,15 @@ def periodic(self) -> None:
                 NEOBrushless.ControlMode.Velocity, constants.kIntakeSpeed * -1
             )
 
+        self.hasPosition = backLimitState or frontLimitState
         SmartDashboard.putNumber(
             constants.kPivotAngleKey, self.pivotEncoder.getPosition().radians()
         )
         SmartDashboard.putNumber(
             constants.kIntakeSpeedKey,
             self.intakeMotor.get(NEOBrushless.ControlMode.Velocity),
         )
+        SmartDashboard.putBoolean(constants.kIntakeHasNoteKey, self.hasPosition)
```

Comment:
Use this in physics instead

Reply:
why would it be in physics? its a part of the intake subsystem

Reply:
See my comment above, where in physics it refers directly to the intake sensor: it should refer to this boolean instead